scripts/test_relcache.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import unittest
```

Comment:
[#990](https://github.com/commaai/opendbc/pull/990), I'm waiting for this PR to merge for opendbc then I'll make a commit to showcase that it passes the test. However, for that commit the only that concerns me is the setting macro as relpath during compilation [can/SConscript](https://github.com/commaai/opendbc/pull/990/files#diff-a5454472c96de11c3576ced37140530a62de48e35ee955729d629cba2aa433af) since it would fail some of the unittest in  opendbc submodule ([fail case](https://github.com/bongbui321/opendbc/actions/runs/7316298646/job/19930787518)) unless I export the BASEDIR such as [this](https://github.com/commaai/opendbc/pull/990/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957f). However, the relpath is needed for openpilot to not rely on absolute path during its compilation